app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -24,3 +24,38 @@ export const NOTES_TABLE_COLUMN_DATA = [
     width: "70%",
   },
 ];
+
+export const NOTES_DATA = [
+  {
+    title: "How to Claim the warranty?",
+    description:
+      "\"Are you getting my texts??? she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting. Of course he was getting them, but if he wasn't getting",
```

Comment:
Why's there a `"\"` at the beginning of the description?

Reply:
Was using \" as an escape sequence in case if we encounter some special keywords/characters in the description.

It should've been like this though:
```
    description:
      "\"Are you getting my texts??? she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting. Of course he was getting them, but if he wasn't getting\",
```